packages/twenty-front/src/modules/object-record/record-table/record-table-cell/components/RecordTableCellFieldContextLabelIdentifier.tsx

```diff
@@ -48,6 +48,7 @@ export const RecordTableCellFieldContextLabelIdentifier = ({
         isLabelIdentifierCompact,
         displayedMaxRows: 1,
         isReadOnly: isFieldReadOnly,
+        maxWidth: columnDefinition.size,
```

Comment:
I believe this is the right fix for now.

Previous implementation was fixing size with css. If we want to go this way, we would need to widthMode for our Chip as Chips are also used in non table (fix width) context. I believe this is the right implementation long term

However, current eng assumptions assume that we have maxWidth available at column level, let's leverage it for now